deploy/terraform/gcp/datastore.tf

```diff
@@ -14,49 +14,53 @@
  * limitations under the License.
  */
 
-resource "google_datastore_index" "blob_status_updated_at" {
-  kind = "blob"
-  properties {
-    name      = "Status"
-    direction = "ASCENDING"
+resource "google_firestore_index" "blob_status_updated_at" {
+  collection  = "blob"
+  query_scope = "COLLECTION_GROUP"
+  api_scope   = "DATASTORE_MODE_API"
+  fields {
+    field_path = "Status"
+    order      = "ASCENDING"
   }
-  properties {
-    name      = "Timestamps.UpdatedAt"
-    direction = "ASCENDING"
+  fields {
```

Comment:
@Zurvarian  These tf files are not used on our end. We need to review our specific TF code and make sure we apply similar updates.